app/Config/core.php

```diff
@@ -74,16 +74,16 @@
  *
  * @see ErrorHandler for more information on exception handling and configuration.
  */
-	Configure::write('Exception', array(
-		'handler' => 'ErrorHandler::handleException',
-		'renderer' => 'ExceptionRenderer',
-		'log' => true
-	));
+    Configure::write('Exception', array(
+        'handler' => 'ErrorHandler::handleException',
+        'renderer' => 'ExceptionRenderer',
+        'log' => true
+    ));
```

Comment:
__[staging][code_sniffer]__
* Closing parenthesis not aligned correctly; expected 34 space(s) but found 4
* Closing parenthesis of a multi-line function call must be on a line by itself